docs/access-management/sso-providers/okta.md

```diff
@@ -1,5 +1,6 @@
 ---
 sidebar_label: Okta
+flavors: oss-enterprise, cloud-teams
```

Comment:
Would `supported_products` might be more self-explanatory to all?
No strong opinion.

Reply:
But also so much to write :sob:

Reply:
products?

Reply:
Sounds good. Will change it to `products` :)